### DISCUSSION TITLE
Boarding - wordkeys request

### DISCUSSION THREAD

**Discussion by lmz** [sheltermanager/asm3#1547] [Q&A / Help] [answered]
Hello - We would like to start using the Boarding feature. A couple questions --
1) I think a Boarding tab only shows up for an animal when they're currently boarding?  Is it possible to see their boarding history?
2) The Boarding Book only shows 5 options: Active, Starting today, Starts in next 3 months, Ending today, and Ends in next 3 months.  It would be useful to be able to go forward and backward more, like a year -- since folks tend to book for the year end holidays way in advance.
3) We would like to implement some boarding forms (Documents) -- a Boarding Cage card, etc. Could we get the boarding information as WordKeys?

Thank you!

**Comment by bobintetley** [comment, marked as answer]
The boarding tab should show up when the animal is eligible for boarding - ie. it's a non-shelter animal or has an open exit movement.

2+3 are good suggestions, I've added a ticket to our dev system for these. There's also a larger ticket for an availability screen for boarding (a bit like the staff rota screen) that should make it more obvious when boarders start and end in a location.

**Comment by lmz** [comment]
Ah - I didn't know about the Non-shelter flag!  Thank you!
